src/containers/MainRouter/index.tsx

```diff
@@ -16,6 +16,14 @@ const SentryRoute = Sentry.withSentryRouting(Route);
 
 const MainRouter: FC<IProps> = () => {
   const location = useLocation();
+  const history = useHistory();
+  useEffect(() => {
+    if (location.pathname === '/vaults') {
+      history.push('/');
```

Comment:
instead of `history.push('/')` , let's add Routes from `constants` file => `history.push(Routes.Invest)`